docs/reference/plotting.rst

```diff
@@ -15,6 +15,7 @@ Gridplot
 Pairplot
 --------
 .. autofunction:: pairplot
+.. autofunction:: pairplot_with_gmm
 
 Degreeplot
```

Comment:
didn't add the ellipse plotting one (or make it a private function)

Reply:
@loftusa I have made it private in the new push coming up.. how to i put a private function in here? or do i not?

Reply:
generally the syntax is `_private_function`, with the underscore before the function
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables

Reply:
Made plot_ellipse() private